base_export_manager/models/ir_exports_line.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2015 Antiun Ingeniería S.L. - Antonio Espinosa
+# Copyright 2015-2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# Copyright 2019 brain-tec AG - Olivier Jossen
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class IrExportsLine(models.Model):
+    _inherit = "ir.exports.line"
+    _order = "sequence,id"
+
+    name = fields.Char(
+        required=False,
+        readonly=True,
```

Comment:
If you set readonly, the inverse method does not write... so chage to False or remove it